src/Stripe.net/Entities/Sources/SourceKlarna.cs

```diff
@@ -0,0 +1,122 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class SourceKlarna : StripeEntity<SourceIdeal>
+    {
+        /// <summary>
+        /// A public URL for a background image, must be served over HTTPS.
+        /// </summary>
+        [JsonProperty("background_image_url")]
+        public string BackgroundImageUrl { get; set; }
+
+        /// <summary>
+        /// Used to initialize the client side Klarna SDK.
+        /// </summary>
+        [JsonProperty("client_token")]
+        public string ClientToken { get; set; }
+
+        /// <summary>
+        /// The first name of the customer (Klarna requires the customer's name
+        /// to be passed as separate first and last name values).
+        /// </summary>
+        [JsonProperty("first_name")]
+        public string FirstName { get; set; }
+
+        /// <summary>
+        /// The last name of the customer.
+        /// </summary>
+        [JsonProperty("last_name")]
+        public string LastName { get; set; }
+
+        /// <summary>
+        /// Format locales as a language tag consisting of a two-letter language
+        /// code combined with a two-letter country code, according to RFC 1766.
+        /// Examples are en-us for U.S. English, en-gb for British English, and
+        /// sv-se for Swedish (in Sweden). (RFC1766).
```

Comment:
Hmmm that one is extremely specific and would never make it to our public docs (or really should not). I'd recommend simplifying the wording here